static-builder-gnu.Dockerfile

```diff
@@ -115,12 +117,27 @@ ENV SPC_LIBC='glibc'
 ENV SPC_CMD_VAR_PHP_MAKE_EXTRA_LDFLAGS_PROGRAM='-Wl,-O3 -pie'
 ENV SPC_CMD_VAR_PHP_MAKE_EXTRA_LIBS='-ldl -lpthread -lm -lresolv -lutil -lrt'
 ENV SPC_OPT_DOWNLOAD_ARGS='--ignore-cache-sources=php-src'
-ENV SPC_OPT_BUILD_ARGS=''
+ENV SPC_OPT_BUILD_ARGS='--with-config-file-path=/etc/frankenphp/php.ini --with-config-file-scan-dir=/etc/frankenphp/php.d'
```

Comment:
Do we really need to change the defaults? If so, we'll have to update the docs too

Reply:
The default for php is `/etc/php.ini` and `/etc/php.d` or `/etc/php/conf.d` depending on the distro. I doubt we wish to use those directories as we'd couple our frankenphp changes to those of the system php package. There are too many reasons why this is a terrible idea.

When compiling php without `--with-config-file-path` it chooses `/usr/local/php/php.ini` and a different additional path depending on the system it was compiled on. That default is not in line with any system installation you would expect from a rpm/deb package. My only other idea for this was `/etc/frankenphp.ini` and `/etc/frankenphp.d`, but that would suggest frankenphp is something different than php, which it isn't.

Reply:
To expand on this, `/usr/local/etc/` is the folder for user, not system installations of self-compiled software. FrankenPHP installed from a package should definitely not reside there - what if the user compiles and installs php themselves? Suddenly our frankenphp package is broken.